### ISSUE TITLE
Display stakesplitthreshold setting

### ISSUE THREAD

**Issue by IlMao-GitHub** [DiviProject/Divi#207]
### Description
With new 2.0.1 the setstakesplitthreshold and getstakesplitthreshold commands have been removed, and now you have to set parm stakesplitthreshold at startup. But you can't display current setting. It could be inserted in getstakingstatus output.

### Environment
N/A

### Notes

N/A

**Comment by IlMao-GitHub** [comment]
This is my proposal:

pi@raspberrypi:~/mydivi/Divi $ git diff
diff --git a/divi/src/rpcmisc.cpp b/divi/src/rpcmisc.cpp
index 80141ca2..fc80f1c3 100644
--- a/divi/src/rpcmisc.cpp
+++ b/divi/src/rpcmisc.cpp
@@ -30,6 +30,7 @@ extern CFeeRate payTxFee;
 #include <spentindex.h>
 #include <net.h>
 #include <txmempool.h>
+#include "Settings.h"

 using namespace boost;
 using namespace boost::assign;
@@ -57,6 +58,7 @@ extern CChain chainActive;
 extern CCriticalSection cs_main;
 extern CTxMemPool mempool;
 extern CFeeRate minRelayTxFee;
+extern Settings& settings;

 bool GetAddressIndex(bool fAddressIndex,
                      CBlockTreeDB* pblocktree,
@@ -1101,6 +1103,7 @@ Value getstakingstatus(const Array& params, bool fHelp)
             "  \"enoughcoins\": true|false,        (boolean) if available coins are greater than reserve balance\n"
             "  \"mnsync\": true|false,             (boolean) if masternode data is synced\n"
             "  \"staking status\": true|false,     (boolean) if the wallet is staking or not\n"
+            "  \"stake split threshold\"           (number)  the set stake split threshold\n"
             "}\n"
             "\nExamples:\n" +
             HelpExampleCli("getstakingstatus", "") + HelpExampleRpc("getstakingstatus", ""));
@@ -1118,6 +1121,10 @@ Value getstakingstatus(const Array& params, bool fHelp)
     bool nStaking = HasRecentlyAttemptedToGenerateProofOfStake();
     obj.push_back(Pair("staking status", nStaking));

+    constexpr char stakeSplitSettingLookup[] = "-stakesplitthreshold";
+    CAmount stakeSplit = static_cast<CAmount>(settings.GetArg(stakeSplitSettingLookup,100000));
+    obj.push_back(Pair("stake split threshold", stakeSplit));
+
     return obj;
 }
 #endif // ENABLE_WALLET

**Comment by IlMao-GitHub** [comment]
`pi@raspberrypi:~/mydivi/Divi $ git diff
diff --git a/divi/src/rpcmisc.cpp b/divi/src/rpcmisc.cpp
index 80141ca2..fc80f1c3 100644
--- a/divi/src/rpcmisc.cpp
+++ b/divi/src/rpcmisc.cpp
@@ -30,6 +30,7 @@ extern CFeeRate payTxFee;
 #include <spentindex.h>
 #include <net.h>
 #include <txmempool.h>
+#include "Settings.h"

 using namespace boost;
 using namespace boost::assign;
@@ -57,6 +58,7 @@ extern CChain chainActive;
 extern CCriticalSection cs_main;
 extern CTxMemPool mempool;
 extern CFeeRate minRelayTxFee;
+extern Settings& settings;

 bool GetAddressIndex(bool fAddressIndex,
                      CBlockTreeDB* pblocktree,
@@ -1101,6 +1103,7 @@ Value getstakingstatus(const Array& params, bool fHelp)
             "  \"enoughcoins\": true|false,        (boolean) if available coins are greater than reserve balance\n"
             "  \"mnsync\": true|false,             (boolean) if masternode data is synced\n"
             "  \"staking status\": true|false,     (boolean) if the wallet is staking or not\n"
+            "  \"stake split threshold\"           (number)  the set stake split threshold\n"
             "}\n"
             "\nExamples:\n" +
             HelpExampleCli("getstakingstatus", "") + HelpExampleRpc("getstakingstatus", ""));
@@ -1118,6 +1121,10 @@ Value getstakingstatus(const Array& params, bool fHelp)
     bool nStaking = HasRecentlyAttemptedToGenerateProofOfStake();
     obj.push_back(Pair("staking status", nStaking));

+    constexpr char stakeSplitSettingLookup[] = "-stakesplitthreshold";
+    CAmount stakeSplit = static_cast<CAmount>(settings.GetArg(stakeSplitSettingLookup,100000));
+    obj.push_back(Pair("stake split threshold", stakeSplit));
+
     return obj;
 }
 #endif // ENABLE_WALLET
`

**Comment by IlMao-GitHub** [comment]
My proposal:

```
pi@raspberrypi:~/mydivi/Divi $ git diff
diff --git a/divi/src/rpcmisc.cpp b/divi/src/rpcmisc.cpp
index 80141ca2..fc80f1c3 100644
--- a/divi/src/rpcmisc.cpp
+++ b/divi/src/rpcmisc.cpp
@@ -30,6 +30,7 @@ extern CFeeRate payTxFee;
 #include <spentindex.h>
 #include <net.h>
 #include <txmempool.h>
+#include "Settings.h"

 using namespace boost;
 using namespace boost::assign;
@@ -57,6 +58,7 @@ extern CChain chainActive;
 extern CCriticalSection cs_main;
 extern CTxMemPool mempool;
 extern CFeeRate minRelayTxFee;
+extern Settings& settings;

 bool GetAddressIndex(bool fAddressIndex,
                      CBlockTreeDB* pblocktree,
@@ -1101,6 +1103,7 @@ Value getstakingstatus(const Array& params, bool fHelp)
             "  \"enoughcoins\": true|false,        (boolean) if available coins are greater than reserve balance\n"
             "  \"mnsync\": true|false,             (boolean) if masternode data is synced\n"
             "  \"staking status\": true|false,     (boolean) if the wallet is staking or not\n"
+            "  \"stake split threshold\"           (number)  the set stake split threshold\n"
             "}\n"
             "\nExamples:\n" +
             HelpExampleCli("getstakingstatus", "") + HelpExampleRpc("getstakingstatus", ""));
@@ -1118,6 +1121,10 @@ Value getstakingstatus(const Array& params, bool fHelp)
     bool nStaking = HasRecentlyAttemptedToGenerateProofOfStake();
     obj.push_back(Pair("staking status", nStaking));

+    constexpr char stakeSplitSettingLookup[] = "-stakesplitthreshold";
+    CAmount stakeSplit = static_cast<CAmount>(settings.GetArg(stakeSplitSettingLookup,100000));
+    obj.push_back(Pair("stake split threshold", stakeSplit));
+
     return obj;
 }
 #endif // ENABLE_WALLET
```

And this is an example of the change:

```
pi@raspberrypi:~/mydivi $ dli getstakingstatus
{
    "validtime" : true,
    "haveconnections" : true,
    "walletunlocked" : true,
    "mintablecoins" : true,
    "enoughcoins" : true,
    "mnsync" : false,
    "staking status" : false,
    "stake split threshold" : 20000
}
```

**Comment by galpHub** [comment]
Hey @IlMao-GitHub 
Just wanted to let you know that this proposal was implemented and is part of the core as of v2.1.0, and will be available in an upcoming release 2.4.0. Thank you for proposing it.

**Comment by IlMao-GitHub** [comment]
Thanks for letting me know! Cheers!
